lib/send-mail.ts

```diff
@@ -117,7 +117,7 @@ export function parseMBoxMessageIDAndReferences(parsed: IParsedMBox):
         if (header.key === "In-Reply-To" || header.key === "References") {
             let value: string = header.value;
             while (value) {
-                const match = value.match(msgIdRegex);
+                const match: RegExpMatchArray | null = value.match(msgIdRegex);
```

Comment:
Is this really needed?

Reply:
same as above..

Reply:
I have updated the PR to remove those changes.